### PR TITLE
New package: genymotion-3.0.4

### PR DESCRIPTION
**Pull request by LinArcX**
#18645

### CODE REVIEW

**Comment by LinArcX** [comment]
What's wrong with license?
I also tired: `license="custom:Proprietary"`, but xlint starts to complain:
> srcpkgs/genymotion/template:13: license 'custom', but no use of vlicense

File: srcpkgs/genymotion/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+do_install() {
+	vmkdir opt
+	vmkdir usr/bin
+	vmkdir usr/share/applications
+	yes | bash ${_full_name} -d $DESTDIR/opt
+	ln -s /opt/genymotion/gmtool $DESTDIR/usr/bin/gmtool
+	ln -s /opt/genymotion/genymotion $DESTDIR/usr/bin/genymotion
+	ln -s /opt/genymotion/player $DESTDIR/usr/bin/genymotion-player
+	ln -s /opt/genymotion/genymotion-shell $DESTDIR/usr/bin/genymotion-shell
+	vinstall ${FILESDIR}/genymotion.desktop 644 usr/share/applications
```
Comment (abenson) [inline srcpkgs/genymotion/template:29]: You need to add this file, it doesn't currently exist

File: srcpkgs/genymotion/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+revision=1
+archs="x86_64"
+_full_name="${pkgname}-${version}-linux_x64.bin"
+create_wrksrc=yes
+build_style=fetch
+hostmakedepends="tar which"
+depends="libpng net-tools protobuf qca-qt5 qt5-script qt5-webkit virtualbox-ose"
+short_desc="Complete set of tools that provides a virtual environment for Android"
+maintainer="linarcx <linarcx@riseup.net>"
+license="custom:Proprietary"
```
Comment (abenson) [inline srcpkgs/genymotion/template:13]: Need to vlicense the file

Comment (LinArcX) [reply srcpkgs/genymotion/template:13]: there's not license file included in downloaded distfile.

Comment (Chocimier) [reply srcpkgs/genymotion/template:13]: It is not redistributable, https://www.genymotion.com/legal/terms-and-conditions/ . Chances to include this are rather low, even as `restricted`.

**Comment by ericonr** [comment]
Closing due to licensing and inactivity. Feel free to reopen.

**Pull request closed without merging**